impeller/renderer/backend/vulkan/render_pass_vk.cc

```diff
@@ -353,20 +358,23 @@ static bool AllocateAndBindDescriptorSets(const ContextVK& context,
       write_set.dstBinding = slot.binding;
       write_set.descriptorCount = 1u;
       write_set.descriptorType = vk::DescriptorType::eCombinedImageSampler;
-      write_set.pImageInfo = &(images[slot.binding] = image_info);
+      FML_DCHECK(image_count < 32u);
+      write_set.pImageInfo = &(images[image_count] = image_info);
```

Comment:
The old implementation would only store one entry if multiple images used the same `slot.binding` index.  Is that behavior desirable?

@chinmaygarde

Reply:
The bindings shouldn't be duplicated, but even if they were the image/buffer map we are processing should have already dedicated (I think)